extensions-core/datasketches/src/test/java/io/druid/query/aggregation/datasketches/quantiles/GenerateTestData.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.aggregation.datasketches.quantiles;
+
+import java.io.BufferedWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Random;
+
+import org.apache.commons.codec.binary.Base64;
+
+import com.yahoo.sketches.quantiles.UpdateDoublesSketch;
+
+public class GenerateTestData
```

Comment:
Hmm, do you think we may need to modify test data someday? It should be rare because we should also fix the expected results in unit tests accordingly.
If you think it's needed, please add some comments on this class to let others know this is used for generating test data for ```DoublesSketchAggregatorTest```.

Reply:
ok